blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+
+Back to PageRank. Everything inside `MAGE` is integrated for you to execute PageRank with ease. By running the following line, you will get results in ~4 seconds for our huge graph. In the other part of the query, we will create and set the `rank` property of the node to a value that the `cugraph.pagerank` algorithm returned under the variable `rank` for every `node`.
+
+
+```python
+  memgraph.execute(
+        """
+        CALL cugraph.pagerank.get() YIELD node,rank
+        SET node.rank = rank;
+        """
+    )
+```
+
+Now, ranks are ready and you can retrieve them with the following Python call:
+
+
+```python
+results =  memgraph.execute_and_fetch(
+        """
+        MATCH (n)
+        RETURN n.id as node, n.rank as rank
+        ORDER BY rank DESC
+        LIMIT 10;
+        """
+    )
+for dict_result in results:
+    print(f"node id: {dict_result['node']}, rank: {dict_result['rank']}")
+```
+
+    node id: 50493, rank: 0.0030278728385218327
+    node id: 31456, rank: 0.0027350282311318468
+    node id: 50150, rank: 0.0025153975342989345
+    node id: 48099, rank: 0.0023413620866201052
+    node id: 49956, rank: 0.0020696403564964
+    node id: 23866, rank: 0.001955167533390466
+    node id: 50442, rank: 0.0019417018181751462
+    node id: 49609, rank: 0.0018211204462452515
+    node id: 50272, rank: 0.0018123518843272954
+    node id: 49676, rank: 0.0014821440895415787
+
+
+Here we only returned 10 nodes with the highest rank score. Results are available in dictionary form. Now it is time to visualize results with **[Memgraph Lab](https://memgraph.com/lab)**. Besides creating beautiful visualizations powered by [D3.js](https://d3js.org/) and our [graph style script](https://memgraph.com/docs/memgraph-lab/graph-style-script-language), you can use **Memgraph Lab** to [query graph database]() and write your graph algorithms in **Python or C++ or even Rust**, check Memgraph Database Logs, visualize graph schema. If you don't have any idea on which dataset you can do it, there are plenty of datasets available for you with **Memgraph Lab** to start and explore. You can learn more about Memgraph Lab [here](https://memgraph.com/docs/memgraph-lab/).
+
+Now, let's find the top 3 Pages and visualize their relationships in the graph. We will use the following query in Memgraph Lab:
```

Comment:
Open Execute Query view in Memgraph Lab and run the following query:

[remember, bearings... some people will look at Memgraph Lab for the first time in their life, help the navigate it]

Reply:
It makes sense